db/forward_iterator.cc

```diff
@@ -95,12 +95,27 @@ class ForwardLevelIterator : public InternalIterator {
     return valid_;
   }
   void SeekToFirst() override {
-    SetFileIndex(0);
```

Comment:
Here, Seek() seeks within the current file, which is set using SetFileIndex() called from the outside. But SeekToFirst() used to work differently and call SetFileIndex(0) for some reason, despite the fact that the caller calls SetFileIndex() before SeekToFirst(), just like it does before Seek(). So I changed SeekToFirst() to work the same way as Seek(). It doesn't affect correctness, just a cleanup.